### PR TITLE
feat(PDS-61):  default theme style

### PR DESCRIPTION
**Pull request by Giogia**
##### [ThemeProvider]

Added a default style initialization, setting font face, font family, weight, size and color

### Issue ticket number and link

[PDS-61](https://makeitapp.atlassian.net/browse/PDS-61?atlOrigin=eyJpIjoiMDRiMzE1ZDljNmRiNGM3ZWI1MzZkYjBhNGU3NWRlNmUiLCJwIjoiaiJ9)

### Checklist

- [x] added the link to the Jira task
- [x] commit message and branch name follow conventions
- [x] tests are included
- [x] changes are accessible and documented from components stories
- [x] typings are updated or integrated accordingly with your changes
- [ ] the changelog has been updated accordingly with your changes
- [x] you are not committing extraneous files or sensible data
- [x] the browser console does not have any logged errors

### CODE REVIEW

File: src/components/ThemeProvider/utils/__tests__/themeDefaultStyle.test.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import themeDefaultStyle from '../themeDefaultStyle'
```
Comment (fredmaggiowski) [inline src/components/ThemeProvider/utils/__tests__/themeDefaultStyle.test.ts:1]: I think the license header is required for tests as well, not only impl files

File: src/components/ThemeProvider/index.tsx
Comment (fredmaggiowski) [inline src/components/ThemeProvider/index.tsx:1]: this file is missing the Apache-2 license header

File: src/components/ThemeProvider/utils/__tests__/themeDefaultStyle.test.ts (hunk 1 above)
Comment (Giogia) [reply src/components/ThemeProvider/utils/__tests__/themeDefaultStyle.test.ts:1]: :+1:

File: src/components/ThemeProvider/index.tsx
Comment (Giogia) [reply src/components/ThemeProvider/index.tsx:1]: I just realized we still miss some other license headers. I added in this PR those ones that are relative to the theme provider. I will open a new one to fix the other files

**Review by Danielecina — APPROVED** [review]

File: src/components/ThemeProvider/index.tsx
Comment (Giogia) [reply src/components/ThemeProvider/index.tsx:1]: https://github.com/mia-platform/design-system/pull/112

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**